assets/oauth-openshift/oauth-serving-cert-configmap.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: oauth-serving-cert
+  namespace: openshift-config-managed
+data:
+  ca-bundle.crt: |-
+{{ include_pki "root-ca.crt" 4 }}
```

Comment:
Please add a new line to this file.  Also you will need to run `make bindata` to update the generated bindata.